library/src/main/java/com/applandeo/materialcalendarview/adapters/CalendarPageAdapter.kt

```diff
@@ -119,6 +121,14 @@ class CalendarPageAdapter(
         calendarGridView.adapter = calendarDayAdapter
     }
 
+    private fun getPageDaysProperties(calendar: Calendar) {
+        val pageCalendarDays = calendarProperties.onPagePrepareListener?.invoke(calendar)
```

Comment:
You can use `?.let {pageCalendarDays -> ... }`to avoid if statement

Reply:
Yeah, I know but for me, it looks better in this case ;)